checks/pytest_.py

```diff
@@ -7,4 +7,8 @@
     args = sys.argv[1:]
     args += ["--exclude", "docs/source/apps/*"]
     args += ["--exclude", "docs/source/optimizations/ibm/ibmq_dd.ipynb"]
+    args += [
+        "--exclude",
+        "client-superstaq/cirq-superstaq/cirq_superstaq/hilbert_daily_integration_test.py",
+    ]
```

Comment:
should qss hilbert_daily_integration_test be excluded as well?

Reply:
Yep. Added.